CORRELATION_RULES_STYLE_GUIDE.md

```diff
@@ -0,0 +1,70 @@
+# panther-analysis Correlation Rule (CR) Style Guide and Best Practices
+
+This style guide highlights essential best practices for writing correlation rules. For a more detailed guide, visit [Correlation Rules](https://docs.panther.com/detections/correlation-rules) in the Panther documentation.
+
+## General advises
+
+- Transition names in rule IDs, and transition IDs should be all caps.
```

Comment:
- comma not necessary
- "be in all-caps"
- this might be just my personal preference, but using "Transition names" here is a little confusing because it makes me wonder whether `Transition.Name` is an [actual Transition field](https://docs.panther.com/detections/correlation-rules/correlation-rule-reference#transitions-fields), like ID. you could rephrase to say, "In a rule ID or transition ID, when describing a transition, put that portion of the ID in all-caps."